docs/Developers/Newcomers_Guide.rst

```diff
@@ -207,6 +209,20 @@ The easy issues that will help you get started are labeled as
 ``difficulty/newcomer`` and are only there to give you a glimpse of what
 it's like to work with us and what the workflow is like.
 
+The coala organisation tries to have ``newcomer`` issues always available
+for people who are joining the organisation.  These newcomer issues are
+intentionally very easy.  The only objective is to give you, the newcomer,
+an opportunity to participate, and learn how to contribute according to
+the coala organisation development guidelines.
+
+coala does **not** follow the GitHub "flow" workflow, so be sure to read
+our guides carefully.  If you do not follow the coala workflow, our team
+will try to nudge you in the right direction.  If nudging isnt effective,
```

Comment:
Also imo the last 3-4 lines are very aggressive and rude. We can instead do `Our community will help you in every way possible to get your PR meged. But still if you can't or you leave it then we'll assign it to someone else.`